spectator-ext-jvm/src/main/java/com/netflix/spectator/jvm/Jmx.java

```diff
@@ -36,15 +35,51 @@ private Jmx() {
    * mbeans from the local jvm.
    */
   public static void registerStandardMXBeans(Registry registry) {
+    monitorClassLoadingMXBean(registry);
+    monitorThreadMXBean(registry);
+    monitorCompilationMXBean(registry);
+
     for (MemoryPoolMXBean mbean : ManagementFactory.getPlatformMXBeans(MemoryPoolMXBean.class)) {
       registry.register(new MemoryPoolMeter(registry, mbean));
     }
-
     for (BufferPoolMXBean mbean : ManagementFactory.getPlatformMXBeans(BufferPoolMXBean.class)) {
       registry.register(new BufferPoolMeter(registry, mbean));
     }
   }
 
+  private static void monitorClassLoadingMXBean(Registry registry) {
+    ClassLoadingMXBean classLoadingMXBean = ManagementFactory.getPlatformMXBean(ClassLoadingMXBean.class);
+    PolledMeter.using(registry)
+      .withName("jvm.classloading.classesLoaded")
+      .monitorMonotonicCounter(classLoadingMXBean, ClassLoadingMXBean::getTotalLoadedClassCount);
+    PolledMeter.using(registry)
+      .withName("jvm.classloading.classesUnloaded")
+      .monitorMonotonicCounter(classLoadingMXBean, ClassLoadingMXBean::getUnloadedClassCount);
+  }
+
+  private static void monitorThreadMXBean(Registry registry) {
+    ThreadMXBean threadMXBean = ManagementFactory.getPlatformMXBean(ThreadMXBean.class);
+    PolledMeter.using(registry)
+      .withName("jvm.thread.startedThreadCount")
+      .monitorMonotonicCounter(threadMXBean, ThreadMXBean::getTotalStartedThreadCount);
+    PolledMeter.using(registry)
+      .withName("jvm.thread.threadCount")
+      .monitorValue(threadMXBean, ThreadMXBean::getThreadCount);
+    PolledMeter.using(registry)
+      .withName("jvm.thread.daemonThreadCount")
+      .monitorValue(threadMXBean, ThreadMXBean::getDaemonThreadCount);
+  }
+
+  private static void monitorCompilationMXBean(Registry registry) {
+    CompilationMXBean compilationMXBean = ManagementFactory.getPlatformMXBean(CompilationMXBean.class);
```

Comment:
`ManagementFactory.getCompilationMXBean()`

Reply:
Oh, I missed there were convenience methods. I'll update the existing memory/buffer references too.